docs/workflows/ios-specific-workflow-steps/xcode-select.md

```diff
@@ -44,18 +44,21 @@ Appcircle provides new versions of Xcode (including beta versions) within 24 hou
 
 :::
 
-:::caution Pool-Based Xcode Version Selection
+### Input Variables
 
-A version other than the Xcode versions on the configuration page should not be entered manually as the Xcode select workflow argument.
-Because the Xcode versions on the configuration page are the versions installed on runners.
-Entering an unavailable Xcode version may cause the build to fail.
-You can review the documentation for detailed information about the Xcode version selection [here](/self-hosted-appcircle/self-hosted-runner/configure-runner/manage-pools#select-pool-for-build-profile).
+This step contains some input variable(s). It needs these variable(s) to work. The table below gives explanation for this variable(s).
 
-:::
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE5421-xcodeSelectInput.png' />
 
-### Input Variables
+:::info Manually Xcode Version Change (Not Recommended)
 
-This step contains some input variable(s). It needs these variable(s) to work. The table below gives explanation for this variable(s).
+In addition to the version change method described above, you can also change the Xcode version **manually**. For this, you can **hard-code** the desired Xcode version into the `$AC_XCODE_VERSION` parameter. For example: `15.1`.
```

Comment:
I think this warning should be under the Version Change heading. This topic fits this heading. (This is what I meant when I said it should replace the warn box that will be deleted in my previous review.)

**If you are going to make this change**, it might be good to specify that ENV is the input:
```suggestion
In addition to the version change method described above, you can also change the Xcode version **manually**. For this, you can **hard-code** the desired Xcode version into the `$AC_XCODE_VERSION` parameter, which serves as the input for the **Xcode Select** step. For example: `15.1`.
```